pkg/quay/quay_debug_test.go

```diff
@@ -231,5 +231,101 @@ func TestCreateNotification(t *testing.T) {
 	if !found {
 		t.Fatalf("Notification %s not found", quayNotification.UUID)
 	}
+}
+
+func TestDeleteNotification(t *testing.T) {
+	if quayToken == "" {
+		return
+	}
+
+	quayClient := NewQuayClient(&http.Client{Transport: &http.Transport{}}, quayToken, quayApiUrl)
+	notification := Notification{
+		Title:  "Test notification",
+		Event:  "repo_push",
+		Method: "webhook",
+		Config: NotificationConfig{
+			Url: "https://example.com",
+		},
+	}
+
+	quayNotification, err := quayClient.CreateNotification(quayOrgName, quayImageRepoName, notification)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if quayNotification == nil {
+		t.Fatal("Notification should not be nil")
+	}
+	if quayNotification.UUID == "" {
+		t.Fatal("Notification UUID should not be empty")
+	}
+
+	res, err := quayClient.DeleteNotification(quayOrgName, quayImageRepoName, quayNotification.UUID)
+	if err != nil || res == false {
+		t.Fatal(err)
+	}
+
+	allNotifications, err := quayClient.GetNotifications(quayOrgName, quayImageRepoName)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	found := false
+	for _, n := range allNotifications {
+		if n.UUID == quayNotification.UUID {
+			found = true
+			break
+		}
+	}
+	if found {
+		t.Fatalf("Notification %s found after deletion", quayNotification.UUID)
+	}
+}
+
+func TestUpdateNotification(t *testing.T) {
```

Comment:
Update, in the implementation, is delete and create. We have cases for both above. I see no sense in this function.